ci/tasks/comment-on-pr-with-pkl-pipeline-diffs.yml

```diff
@@ -0,0 +1,187 @@
+---
+platform: linux
+image_resource:
+  type: registry-image
+  source:
+    repository: governmentdigitalservice/pay-concourse-runner
+params:
+  GITHUB_TOKEN:
+  CONCOURSE_TEAM:
+  FLY_PASSWORD:
+inputs:
+  - name: pay-ci
+  - name: pkl-pipeline-pr
+outputs:
+  - name: diffs
+run:
+  path: ash
+  args:
+    - -c
```

Comment:
Not a blocker, but would it be worth moving the script to a separate .sh file to get IDE support when editing or for linting?